src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  --normal-color: black;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  background-color: white;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
```

Comment:
delete this, and add `align-items: center;` to `.header` to center all ur elemets by `y axixs`
```suggestion
```